README.md

```diff
@@ -1 +1,33 @@
-# github-flow
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# github-flow
+
+> This is an example of github flow. Add new feature branch and make commits.
+
+
+## Built With
+
+- Js
+- Atom Git
+
+
+## Authors
+
+👤 **Alejandro Maggioni**
+
+- GitHub: [@githubhandle](https://github.com/alex1779/)
+- Twitter: [@twitterhandle](https://twitter.com/alex1779)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/alejandro-maggioni-086678b5/)
+
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```

Comment:
[OPTIONAL] You can take this note out because it serves as an instruction to guide you hence It is not needed in the readme